js/components/unmuteIcon.js

```diff
@@ -0,0 +1,77 @@
+var React = require('react'),
+    Icon = require('./icon'),
+    ClassNames = require('classnames'),
+    CONSTANTS = require('../constants/constants');
+
+var UnmuteIcon = React.createClass({
+  getInitialState: function () {
+    return {
+      collapseTime: 2000,
+      expanded: !this.props.controller.state.volumeState.unmuteIconCollapsed
```

Comment:
Sorry, one last thing I forgot to mention. Usually it's not recommended to assign values from props to local state. We can probably avoid the local state by using `unmuteIconCollapsed` directly when rendering. We can probably change this later though

Reply:
I'll make this change now.

Reply:
Done